lib/devise-security/models/old_password.rb

```diff
@@ -1,6 +0,0 @@
-# frozen_string_literal: true
-
-require 'active_record'
-class OldPassword < ActiveRecord::Base
```

Comment:
I don't understand why this is removed.

Reply:
Moved to app, because i can't verify mongoid/activerecord after inheritance, but I can verify and generate right model structure in the installation

Reply:
I'll let the other maintainers weigh in but that doesn't fly with me. It makes it much harder for users to use this gem out of the box and moves this model out of our control if we want to make changes. We've discussed splitting this gems out into smaller gems and maybe after that point there can be  `password-archivable` and `password-archivable-mongoid` as separate gems.

Reply:
I agree, much better.

Personally I prefer the activerecord but where I work they use mongoid and I was forced to create this portability, then I decided to do the PR to collaborate with others in my situation but I agree with you that having to create hacks in the code for this is bad.